api-src/org/labkey/api/targetedms/BlibSourceFiles.java

```diff
@@ -0,0 +1,23 @@
+package org.labkey.api.targetedms;
+
+public class BlibSourceFiles
+{
+    private String[] spectrumSourceFiles;
```

Comment:
Can these be lists instead of arrays? I see there's code that's converting them when calling the constructor.